grpc-bridge/ResteasyGrpcProviderFactoryImpl.java

```diff
@@ -0,0 +1,17 @@
+package org.jboss.resteasy.grpc.protobuf;
```

Comment:
This looks strange. The path and package do not match.

Reply:
I just moved it out of the source tree instead of deleting it. Gone.